contracts/MechMarketplace.sol

```diff
@@ -233,7 +260,7 @@ contract MechMarketplace is IErrorsMarketplace {
         mechDelivery.requester = msg.sender;
```

Comment:
why not store the payment here instead of the new mapping?
mechDelivery.payment = msg.value;

Reply:
oh great, of course, so easy :O